solution-base/kubedb/kubedb-catalog-mongodb.yaml

```diff
@@ -0,0 +1,602 @@
+---
```

Comment:
since we're managing this file, we can probably just create one version for the version we use. This would reduce the line changes.